client/src/pages/AboutPage/components/Affiliate.jsx

```diff
@@ -0,0 +1,10 @@
+const Affiliate = ({ image, name }) => {
+	return (
+		<div className="affiliate">
+			<img src={image} alt="circle" className="circle_image" />
+			<p>{name}</p>
+		</div>
+	);
+};
```

Comment:
The "circle_image" class is not mentioned anywhere in the stylesheets
Also the class naming should make sense in context

Also for accessibility we could pass in the alt text as a prop as well (and store it in the same object we provide thename and image source) see below

```suggestion
const Affiliate = ({ name, image, altText }) => {
	return (
		<div className="affiliate-image-container"> // this classname is adjusted - but its up to you
			<img src={image} alt={altText} className="affiliate-image" /> // this classname is adjusted - but its up to you
			<p>{name}</p>
		</div>
	);
};
```